src/chain.h

```diff
@@ -531,6 +540,13 @@ class CDiskBlockIndex : public CBlockIndex
                 READWRITE(lelantusMintedPubCoins);
             READWRITE(lelantusSpentSerials);
 
+            if (!(s.GetType() & SER_GETHASH)
+                && nHeight >= params.nSparkStartBlock) {
+                READWRITE(sparkMintedCoins);
+                READWRITE(sparkSetHash);
+                READWRITE(spentLTags);
```

Comment:
Serialization of these to index won't fully work without copying it explicitly in `LoadBlockIndexFromGuts` in `txdb.cpp`

Reply:
good catch, fixed.